docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc

```diff
@@ -4,7 +4,9 @@
 include::{fleet-repo-dir}/standalone-note.asciidoc[]
 
 Standalone {agent}s are manually configured and managed locally on the systems
-where they are installed. To configure standalone {agent}s, specify settings
+where they are installed. They are useful when you are not interested in centrally managing agents in Fleet, either due to your company's security requirements, or because you prefer to use another configuration management system.
```

Comment:
```suggestion
where they are installed. They are useful when you are not interested in centrally managing agents in {fleet}, either due to your company's security requirements, or because you prefer to use another configuration management system.
```